charts/big-peer/values.yaml

```diff
@@ -72,15 +73,15 @@ ingress:
 
     # -- Set the ingressClass that is used for this ingress.
     # Requires Kubernetes >=1.19
-    ingressClassName:  # "traefik"
+    ingressClassName: alb # "traefik"
```

Comment:
This might be just a miss but we should assign a default here as the ingress is disabled by default therefore we can't assume a class name.